GOVERNANCE.md

```diff
@@ -0,0 +1,111 @@
+# Keycloak Governance
+
+* [Vision](#vision) 
+* [Maintainers](#maintainers) 
+* [Contributing](#contributing)
+
+## Vision
+
+Keycloak aims to be easy to use and lightweight. The project was founded to make it easy for application developers 
+to secure modern applications and services.
+
+The 80/20 rule is a core part of the vision behind Keycloak. Stating that 80% of requirements come from roughly 20%
+of the use cases. We strongly believe if Keycloak would support all use cases by default it would become bloated 
+and hard to use.
+
+Keycloak aims to be opinionated and make it as easy as possible to achieve the common use cases, while still
+enabling the less common use cases through custom extensions.
+
+
+## Projects
+
+Keycloak consists of several projects:
+
+* [Keycloak](https://github.com/keycloak/keycloak) - Keycloak Server and Java adapters
+* [Keycloak Documentation](https://github.com/keycloak/keycloak-documentation) - Documentation for Keycloak
+* [Keycloak QuickStarts](https://github.com/keycloak/keycloak-quickstarts) - QuickStarts for getting started with Keycloak
+* [Keycloak Containers](https://github.com/keycloak/keycloak-containers) - Container images for Keycloak
+* [Keycloak Gatekeeper](https://github.com/keycloak/keycloak-gatekeeper) - Proxy service to secure apps and services with Keycloak
+* [Keycloak Node.js Connect](https://github.com/keycloak/keycloak-nodejs-connect) - Node.js adapter for Keycloak
+* [Keycloak Node.js Admin Client](https://github.com/keycloak/keycloak-nodejs-admin-client) - Node.js library for Keycloak Admin REST API
+
+The same governance model applies to all projects. However, the list of maintainers may vary per project. 
+
+
+## Maintainers
+
+The list of maintainers can be found in the [MAINTAINERS.md](MAINTAINERS.md) file in the repository for the individual 
+projects listed in the [Projects](#projects) section.
+
+### Maintainer Responsibilities
+
+A maintainer is someone that has shown deep knowledge of vision, features and codebase. It is their 
```

Comment:
should be "someone who"  -  use "that" when not referring to a person

Reply:
Combined to a single sentence "The 80/20 rule, that states 80% of requirements come from around 20% of use cases, is a core part of the vision behind 
Keycloak. "

Reply:
Fixing